src/dotnet-core-uninstall/Shared/Configs/CommandLineConfigs.cs

```diff
@@ -227,7 +228,9 @@ static CommandLineConfigs()
             DryRunCommand.Handler = CommandHandler.Create(ExceptionHandler.HandleException(() => DryRunCommandExec.Execute()));
             RemoveCommand.Handler = CommandHandler.Create(ExceptionHandler.HandleException(() => UninstallCommandExec.Execute()));
 
-            CommandLineParseResult = UninstallRootCommand.Parse(Environment.GetCommandLineArgs());
+            var parser = new CommandLineBuilder(UninstallRootCommand)
```

Comment:
Why this change? `CommandLineBuilder(UninstallRootCommand)` should be the default.

Reply:
I'm working on customizing the help builder right now which will require a parser build from the command line builder. I didn't mean to include this change in this commit, but it will be part of the next PR.